evap/settings.py

```diff
@@ -352,6 +352,11 @@ class ManifestStaticFilesStorageWithJsReplacement(ManifestStaticFilesStorage):
 # Absolute filesystem path to the directory that will hold user-uploaded files.
 MEDIA_ROOT = os.path.join(BASE_DIR, "upload")
 
+### Evaluation progress rewards
+GLOBAL_EVALUATION_PROGRESS_REWARDS: list[tuple[float, str]] = []  # (required_voter_ratio between 0 and 1, reward_text)
```

Comment:
Code only accesses it as pairs, we don't need key-lookup, so we don't need the hashing part of the dictionary. Fair, a dictionary cannot contain the same key twice, and we don't expect two rewards with the same ratio.

Both, list and dict, are fine with me. Preferences on your side?

Reply:
Multiplicity is not too much of a modelling concern for me, my only real criticism is that the list carries an ordering that is never used - dict feels more natural to me in that regard

Im the end both are totally fine

Reply:
Sticking with list for now, simply because it is "the" default container. Yes it has ordering, but a dict has as well. We could switch to an unordered_set (don't know if python `set` keeps insertion order like `dict`?)